### PR TITLE
Update transfer-a-domain-from-microsoft-to-another-host.md

### PR DESCRIPTION
**Pull request by msbemba**
Added line about signing in as GA

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9103

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/get-help-with-domains/transfer-a-domain-from-microsoft-to-another-host.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ You can't transfer a Microsoft 365 domain to another registrar for 60 days after
 > [!NOTE]
 > A _Whois_ query shows a Microsoft purchased domain registrar as Wild West Domains LLC. However, only Microsoft should be contacted regarding your Microsoft 365 purchased domain.
 
-Follow these steps to get a code at Microsoft 365, and then go to the other domain registrar website to set up transferring your domain name to the new registrar.
+Sign in as a Global Administrator, follow these steps to get a code at Microsoft 365, and then go to the other domain registrar website to set up transferring your domain name to the new registrar.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/transfer-a-domain-from-microsoft-to-another-host.md]:
```suggestion
Sign in as a global administrator, follow these steps to get a code at Microsoft 365, and then go to the other domain registrar website to set up transferring your domain name to the new registrar.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, I know there are many inconsistencies in the capitalization of global admin, but this is the latest guidance. Thanks.

**Comment by yogkumgit** [comment]
@msbemba Gentle reminder

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@efrene Please merge this PR. Thanks

**Pull request merged**